samples/spookyshots/main.py

```diff
@@ -0,0 +1,139 @@
+import streamlit as st
+from streamlit_option_menu import option_menu
+import cloudinary
+from cloudinary import CloudinaryImage
+import cloudinary.uploader
+import cloudinary.api
+from dotenv import load_dotenv
+import os
+
+load_dotenv()
+
+cloudinary.config(
+    cloud_name=os.getenv('CLOUDINARY_CLOUD_NAME'),
```

Comment:
This step is redundant, SDK will consume these variables by default. (these are the actual variable names)

Reply:
Can you tell me more about this?

Reply:
@codingis4noobs2 , sure:
https://github.com/cloudinary/pycloudinary/blob/89006f293051cd403fc97920822446267aba7d0c/cloudinary/__init__.py#L203

When SDK is initialized, it checks for environment variables and consumes them.

You can use 2 forms: 1. Use single `CLOUDIANRY_URL` (that you can copy from the console) or use variable per value(as you used it).

Reply:
Hey @const-cloudinary , I tried this code:
```py
cloudinary.config(
    secure=True
)
```
as far as i understand sdk will try to get CLOUDINARY_CLOUD_NAME or CLOUDINARY_URL, and if CLOUDINARY_CLOUD_NAME is present, it will also get other variables that starts with CLOUDINARY_, and convert them to something like api_key. When I tried this, I got an error that API key is missing. Need some guidance on this.

Reply:
@codingis4noobs2 the code is running whenever you import module. Since you call `load_dotenv()` after, there is no effect.

You can run `cloudinary.reset_config()`  after `load_dotenv()`.

Reply:
This worked 🚀